article.go

```diff
@@ -0,0 +1,68 @@
+package goshopify
+
+import (
+	"fmt"
+)
+
+const articlesBasePath = "articles"
+const articleCountBasePath = "articles/count"
+
+// ArticleService is an interface for interfacing with the article endpoints
+// of the Shopify API.
+// See: https://help.shopify.com/api/reference/article
```

Comment:
Can you please leave a TODO comment that this is incomplete and that there are more Operations that still need to be added?